File: Find the Discount/FindDiscount.py
def dis(OgPrice, discount):
    return round(OgPrice * ((100 - discount) / 100), 2)


# using challenge's examples
print(dis(1500, 50))
print(dis(89, 20))
print(dis(100, 75))

# for testing other numbers
OgPrice = float(input("Original price: "))
Discount = float(input("Discount: "))

print(f"Discounted price: {dis(OgPrice, Discount)}.")

File: Find the Discount/FindDiscount.cpp
#include <iostream>

double dis(double OgPrice, double discount)
{
    return OgPrice * ((100 - discount) / 100);
}

/*int main()
{
    double OgPrice, discount;
    std::cout << "Original price: ";
    std::cin >> OgPrice;
    std::cout << "Discount: ";
    std::cin >> discount;

    std::cout << "Discounted price: " << dis(OgPrice, discount) << ".\n";

    return 0;
}*/